include/grpc++/support/auth_context.h

```diff
@@ -72,20 +72,26 @@ class AuthPropertyIterator
   const char* name_;
 };
 
+/// Class encapsulating the Authentication Information.
+/// 
+/// It includes the secure identity of the peer, the type of secure transport
+/// used as well as any other properties required by the authorization layer.
```

Comment:
Actually, nevermind, done in this one. Please take a look. There was the option of having them as functions outside of classes but I made them static methods of AuthContext to have them inside the class' namespace.